<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Preferences</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>OBS Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="OBSAddr"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>OBS Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="OBSPort">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>OBS Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="OBSPass"/>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>OBSAddr</tabstop>
  <tabstop>OBSPort</tabstop>
  <tabstop>OBSPass</tabstop>
  <tabstop>SaveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
